ADALiOS/ADALiOS/ADAuthenticationWebViewController.m

```diff
@@ -215,6 +215,19 @@ - (void)webView:(UIWebView *)webView didFailLoadWithError:(NSError *)error
         return;
     }
     
+    // If we failed on an invalid URL check to see if it matches our end URL
+    if ([error.domain isEqualToString:@"NSURLErrorDomain"] && (error.code == -1002 || error.code == -1003))
```

Comment:
@RPangrle  Can you define constants for these error codes that have a descriptive name?